src/main/java/service/impl/ReportCreateServiceImpl.java

```diff
@@ -0,0 +1,27 @@
+package service.impl;
+
+import dao.StorageDao;
+import java.util.stream.Collectors;
+import service.ReportCreateService;
+
+public class ReportCreateServiceImpl implements ReportCreateService {
+    private static final String SPLITERATOR = ",";
+    private static final String TITLE = "fruit,quantity";
+    private final StorageDao storageDao;
+
+    public ReportCreateServiceImpl(StorageDao storageDao) {
+        this.storageDao = storageDao;
+    }
```

Comment:
I think we could also pass header as input parameter, so we can use this service for reports with different headers

Reply:
Could you explain, please? What does header mean?